engine.py

```diff
@@ -379,6 +379,38 @@ def generate_thumbnail(self, document=None, output_path=None):
 
             return output_path
 
+    def save_as(self, document):
+        """
+        Launch a Qt file browser to select a file, then save the supplied
+        document to that path.
+
+        :param document: The document to save.
+        """
+
+        from sgtk.platform.qt import QtGui
+
+        try:
+            doc_path = document.fullName.fsName
```

Comment:
Use this instead:

https://github.com/shotgunsoftware/tk-photoshopcc/blob/master/python/tk_photoshopcc/adobe_bridge.py#L175